CONTRIBUTING.md

```diff
@@ -1 +1,39 @@
-Contributors transfer all ownership rights to ZTI upon acceptance of their contributions into the primary ZTI repository. Further, contributor's guarantee that they have the right to transfer ownership of their contributions; and shall indemnify ZTI from any and all repercussions caused by a violation of this guarantee.
\ No newline at end of file
+# Principles
+1. Whoever writes the code, choses the license
+2. Collective action, Mutual accountability
+3. Don't do or say anything that you'd want your mother to know
+
+# Why the Legal bovine excrement
+Because this interfaces financial payment system, Uncle Sam needs 
+1. Regulatory compliance with pertinent regulations [TBC title&section], now if this was just DIY it'd be an MIT license but since one of the service tiers is a hosted FaaS, someone (guess who) is going to be on the hook for leaky code
+2. Any suspicious activity of individual transaction >$10k needs to be reported to [Financial Crimes Enforcement Network](https://www.fincen.gov), goodbye lunchtime
+3. Having responsibility without legal control is like carrying a gun with safety off, do you want to take over this project?
```

Comment:
I like the framing of these quite a bit! I'm trying to think about how we can shrink it down and make the language tighter and more approachable; and I wonder if the appropriate place would be a FAQ section  underneath the [legal heading](https://github.com/zinc-collective/compensated/blob/0.X/CONTRIBUTING.md#legal) in the current verison.

Reply:
I assign all rights, including copyright, in https://github.com/zinc-collective/compensated/*.[md,html,]? and /core to Zinc Collective LLC.